akka-remote/src/main/resources/reference.conf

```diff
@@ -20,6 +20,8 @@ akka {
 
     serialization-bindings {
       "akka.actor.ActorSelectionMessage" = akka-containers
+      "akka.actor.Identify" = akka-containers
+      "akka.actor.ActorIdentity" = akka-containers
```

Comment:
+1 to comment and I'd also want a ticket so we don't forget to do the change when breaking compat